apphub/multi_task_learning/uncertainty_weighted_loss/uncertainty_loss.ipynb

```diff
@@ -19,7 +19,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 10,
```

Comment:
we typically restart the kernel so that the execution count doesn't display weird for people

Reply:
sorry for the confusion, I didn't mean not to run the notebook at all. I was trying to say to restart the kernel + run the notebook such that the cells are executed in order starting from 1